test/index-test.js

```diff
@@ -26,7 +26,7 @@ describe('Server', () => {
         expect(response.statusCode).equal(expectedStatus);
     });
 
-    it(`Sends the correct html`, async () => {
+    /*it(`Sends the correct html`, async () => {
```

Comment:
Never use comments to alter your app or test flow. Since we use git, you can just delete it. An another option for tests - use `describe.skip` or `it.skip`.

Reply:
BTW, it's still a valid test. You just have to change assertion, e.g. assert the title  or/and a link to your JS bundle.